pkg/writer/sarif.go

```diff
@@ -103,7 +104,8 @@ func writeSarif(data interface{}, writer io.Writer, forGithub bool) error {
 	}
 
 	// print the report to anything that implements `io.Writer`
-	return report.PrettyWrite(writer)
+	// Sarif already writes the result in a file hence we are ignoring the file output writer
```

Comment:
so for sarif writer the output file won't be present in the provided log output directory?

Reply:
added support for sarif as well.